STACpopulator/extensions/cmip6.py

```diff
@@ -118,7 +132,7 @@ def apply(self, attrs: Dict[str, Any]) -> None:
             variables : Dictionary mapping variable name to a :class:`Variable`
                 object.
         """
-        self.properties.update(**Properties(**attrs).model_dump_json())
+        self.properties.update(**Properties(**attrs).model_dump())
```

Comment:
This one could be a problem, because the json serializer from pystac does not know how to serialize some of the content.